src/OpenTelemetry.Instrumentation.EntityFrameworkCore/Implementation/EntityFrameworkDiagnosticListener.cs

```diff
@@ -40,6 +41,8 @@ internal sealed class EntityFrameworkDiagnosticListener : ListenerHandler
     private readonly PropertyFetcher<CommandType> commandTypeFetcher = new("CommandType");
     private readonly PropertyFetcher<string> commandTextFetcher = new("CommandText");
     private readonly PropertyFetcher<Exception> exceptionFetcher = new("Exception");
+    private readonly PropertyFetcher<string> hostFetcher = new("Host");
+    private readonly PropertyFetcher<string> portFetcher = new("Port");
```

Comment:
https://github.com/open-telemetry/semantic-conventions/blob/v1.30.0/docs/database/database-spans.md

port should be int not string.